### PR TITLE
Add left_handed option for tablets

### PR DESCRIPTION
**Pull request by khalid151**
#### Describe your PR, what does it fix/add?
Adding `left_handed` option for tablets allow them to be rotated 180 degrees.
I've had issues with using transform with my tablet, and `libinput_device_config_left_handed_set` was missing.
Adding this options fixed the rotation issue I had. 

#### Is it ready for merging, or does it need work?
Yes.

### CODE REVIEW

File: src/managers/input/InputManager.cpp [hunk 1]
```diff
@@ -1418,6 +1418,11 @@ void CInputManager::setTabletConfigs() {
             Debug::log(LOG, "Setting calibration matrix for device {}", t.name);
             libinput_device_config_calibration_set_matrix(LIBINPUTDEV, MATRICES[ROTATION]);
 
+            if (g_pConfigManager->getDeviceInt(t.name, "left_handed", "input:left_handed") == 0)
```
Comment (vaxerski) [inline src/managers/input/InputManager.cpp:1421]: this should go to input:tablet:

**Review by vaxerski — COMMENTED** [review]
I thought left-handed people didn't exist and it was a conspiracy, dam

File: src/managers/input/InputManager.cpp (hunk 1 above)
Comment (khalid151) [reply src/managers/input/InputManager.cpp:1421]: Oops, my bad. I'll update it

**Review by vaxerski — APPROVED** [review]
lgtm, wiki mr pls

**Comment by khalid151** [comment]
Done
https://github.com/hyprwm/hyprland-wiki/pull/544

**Pull request merged**